### PR TITLE
Added contract status for jobs list and single gop post + button to e…

### PR DESCRIPTION
**Pull request by DavidSurkov**
image
image

### CODE REVIEW

File: src/hooks/use-modal-handler.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (milkavladislav) [inline src/hooks/use-modal-handler.ts]: IMHO u should use default export and there will be no need to use eslint-disable-next-line

**Review by milkavladislav — CHANGES_REQUESTED** [review]
small remark

File: src/hooks/use-modal-handler.ts (hunk 1 above)
Comment (DavidSurkov) [reply src/hooks/use-modal-handler.ts]: or we can just remove this lint setting

Comment (DavidSurkov) [reply src/hooks/use-modal-handler.ts]: done

**Review by milkavladislav — APPROVED** [review]

File: src/components/UI/JobItems/JobItems.tsx [hunk 2]
```diff
@@ -32,6 +36,7 @@ function JobItems({
       <JobTitle>
         <RouterLink to={link}>{title}</RouterLink>
         <DeatailedInfo>
+          <ContractStatus>{contractStatus || "Pending"}</ContractStatus>
```
Comment (lyudad) [inline src/components/UI/JobItems/JobItems.tsx]: "Pending" - if it status move to status constant

File: src/pages/JobOwner/MyJobs/ClientJob/ClientJob.tsx [hunk 3]
```diff
@@ -21,8 +24,19 @@ function ClientJob(): JSX.Element {
...
+  const { modal, toggleModal } = useModalHandler();
+
   return (
     <PageContainer>
+      <Modal
+        visible={modal}
+        title={t("MyJobs.endContractTitle")}
+        onOk={toggleModal}
+        onCancel={toggleModal}
+        okText="Continue"
```
Comment (lyudad) [inline src/pages/JobOwner/MyJobs/ClientJob/ClientJob.tsx]: also through t()

**Pull request merged**